io-uring-test/src/tests/register.rs

```diff
@@ -0,0 +1,70 @@
+use crate::Test;
+use io_uring::{cqueue, opcode, squeue, IoUring};
+
+pub fn test_register_files_sparse<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
```

Comment:
same

Reply:
When I run *cargo test*, it passes. I guess something is adding the *unstable* feature flag when the test is run, or perhaps all feature flags are added? How do I run the test without the *unstable* feature?